apps/eth_blockchain/contracts/ERC20/README.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This docs is pure gold.

Reply:
Credits to https://github.com/OpenZeppelin/openzeppelin-contracts :)